plugin/converter/jsonnet.go

```diff
@@ -19,14 +20,16 @@ import (
 
 // Jsonnet returns a conversion service that converts the
 // jsonnet file to a yaml file.
-func Jsonnet(enabled bool) core.ConvertService {
+func Jsonnet(enabled bool, client *scm.Client) core.ConvertService {
```

Comment:
We already have a service: https://github.com/drone/drone/blob/master/service/content/content.go#L49 that can be used rather than injecting the scm.Client directly and does all the heavy lifting for you.

Reply:
Thanks, changed to use FileService which also closed the open question about authentication tokens.